### PR TITLE
Set Service Delivery defaults based on fields from selected service field set value

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
~~Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage~~
~~CRUD/FLS is enforced in Apex~~
~~Permission sets are updated to account for CRUD|FLS|Tab|Classes~~
~~Field sets are updated to account for new fields~~
- [ ] UX approval or UX not necessary
- [ ] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
~~PR contains draft release notes~~
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by jjbennett** [comment]
[W-8921775](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000097DGSIA2/view)

File: force-app/main/default/lwc/groupServiceDelivery/groupServiceDelivery.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        PROGRAM_ENGAGEMENT_FIELD.fieldApiName,
+        this.serviceField,
+    ];
+    labels = { selectService };
+
+    @wire(getRecord, {
+        recordId: "$serviceId",
+        fields: [BULK_SERVICE_DELIVERY_FIELD_SET_FIELD],
+    })
+    wiredSession(result) {
```
Comment (AndersonTarren) [inline force-app/main/default/lwc/groupServiceDelivery/groupServiceDelivery.js:35]: Curious about the method name.  Appears to get a getRecord against a service - would expect wiredService.

**Review by AndersonTarren — APPROVED** [review]
Looks and works well.  I called out one question about a method name.  I also see that I'm able to proceed without selecting a Service, which is flagged as required.

File: force-app/main/default/lwc/groupServiceDelivery/groupServiceDelivery.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<template>
+    <div class="slds-box">
+        <lightning-record-edit-form object-api-name={objectApiName} density="comfy">
```
[the remark this answers is not in the document]
Comment (jjbennett) [reply force-app/main/default/lwc/groupServiceDelivery/groupServiceDelivery.html:3]: I fixed the required fields in the next iteration as well as repopulating on back.

File: force-app/main/default/lwc/groupServiceDelivery/groupServiceDelivery.js (hunk 1 above)
Comment (jjbennett) [reply force-app/main/default/lwc/groupServiceDelivery/groupServiceDelivery.js:35]: copy and paste error lol... I will fix on the next iteration

**Pull request merged**